examples/IMDb/leap_custom_test.py

```diff
@@ -0,0 +1,122 @@
+from leap_binder import *
+import tensorflow as tf
+import os
+import numpy as np
+import onnxruntime
+from keras.losses import BinaryCrossentropy
+
+def check_custom_test():
+    print("started custom tests")
+    responses = preprocess_func()
+    train = responses[0]
+    val = responses[1]
+    responses_set = train
+    dir_path = os.path.dirname(os.path.abspath(__file__))
+    model_path = 'model/fabriceyhc-bert-base-uncased-imdb.onnx'
```

Comment:
is this onnx a part of the repo? if not maybe call cloud here to get it?

Reply:
uploaded to the repo